packages/playground/Samples/focusZone.tsx

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ * @format
+ */
+
+import * as React from 'react';
+import {
+  AppRegistry,
+  Button,
+  StyleSheet,
+  TextInput,
+  Switch,
+  View,
+  Text,
+  requireNativeComponent,
+  FlatList,
+} from 'react-native';
+
+const FocusZoneView = requireNativeComponent('FocusZoneView');
```

Comment:
Seems to me we should have a FocusZoneView.tsx as part of this PR so end users don't need to use `requireNativeComponent`?

Reply:
True, the current state of the PR is straddled between two implementations, so I may have missed some bits when moving between the two. Good call.